app/views/stats/index.html.erb

```diff
@@ -24,12 +24,18 @@
 <div class="wrapper">
   <dl class="statistics meta group" role="complementary">
     <% [:user_subscriptions, :kudos, :comments, :bookmarks, :subscriptions, :word_count].each do |stat| %>
-      <dt><%= (stat.to_s.titleize == "Comments") ? ts("Comment Threads") : stat.to_s.titleize %></dt>
-      <dd><%= @totals[stat] %></dd>
+      <% if stat == :word_count
+          html_class = 'words'
+        else
+          html_class = stat.to_s.humanize.downcase
+        end
+      %>
+      <dt class="<%= html_class %>"><%= (stat == :comments) ? ts("Comment Threads:") : stat.to_s.titleize %></dt>
```

Comment:
This `stat.to_s.titleize` will probably come back to bite us when we do start translating things but the alternative (a massive if statement) is too horrible to contemplate. _lifts carpet and sweeps_